sparkmonitor/__init__.py

```diff
@@ -18,11 +19,15 @@ def _jupyter_nbextension_paths():
         # directory in the `nbextension/` namespace
         dest='sparkmonitor',
         # _also_ in the `nbextension/` namespace
-        require='sparkmonitor/module')]
+        require='sparkmonitor/extension')]
 
+HERE = Path(__file__).parent.resolve()
 
-def _jupyter_server_extension_paths():
```

Comment:
Yes, I've removed the server extension that does the proxying entirely.

It is currently disabled in the code in the production version we have as it only works for local spark. 

I will explore using https://github.com/jupyterhub/jupyter-server-proxy or a more generic solution to proxy things running inside the session, or if not I will add it back with the necessary changes as a separate PR.

Reply:
Ok! 
I have some doubts `jupyter-server-proxy` does what we want, since I believe it doesn't rewrite the content to update the urls and only translates a url to a port and nothing else. But check it ofc.